agenta-web/src/components/AppSelector/Welcome.tsx

```diff
@@ -81,85 +75,89 @@ interface Props {
 const Welcome: React.FC<Props> = ({onCreateAppClick}) => {
     const {appTheme} = useAppTheme()
     const classes = useStyles({themeMode: appTheme} as StyleProps)
+    const {Text} = Typography
     const isDemo = process.env.NEXT_PUBLIC_FF === "demo"
```

Comment:
Not specific to this PR. But can you please create a reusable constant inside at the path `src/constants/enviornments.ts` , that wraps this logic?

```javascript
export const IS_DEMO_ENV = process.env.NEXT_PUBLIC_FF === "demo"
```
And then use it here. Would be nice to also replace it everywhere else in the codebase

Would be a worthwhile refactoring